README.md

```diff
@@ -81,6 +81,7 @@
 | [kubernetes-dashboard-metrics-scraper](./images/kubernetes-dashboard-metrics-scraper) | `cgr.dev/chainguard/kubernetes-dashboard-metrics-scraper` | stable | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/kubernetes-dashboard-metrics-scraper.build.status.latest-dev.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/kubernetes-dashboard-metrics-scraper:latest-dev)<br/>[![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/kubernetes-dashboard-metrics-scraper.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/kubernetes-dashboard-metrics-scraper:latest) |
 | [kubernetes-ingress-defaultbackend](./images/kubernetes-ingress-defaultbackend) | `cgr.dev/chainguard/kubernetes-ingress-defaultbackend` | stable | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/kubernetes-ingress-defaultbackend.build.status.latest-dev.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/kubernetes-ingress-defaultbackend:latest-dev)<br/>[![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/kubernetes-ingress-defaultbackend.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/kubernetes-ingress-defaultbackend:latest) |
 | [kustomize-controller](./images/kustomize-controller) | `cgr.dev/chainguard/kustomize-controller` | experimental | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/kustomize-controller.build.status.latest-dev.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/kustomize-controller:latest-dev)<br/>[![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/kustomize-controller.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/kustomize-controller:latest) |
+| [kyverno-cli](./images/kyverno-cli) | `cgr.dev/chainguard/kyverno-cli` | stable | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/kyverno-cli.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/kyverno-cli:latest) |
```

Comment:
unclear right now why only this one showing up in the readme

Reply:
Yes, have been trying to figure it out but not sure what could be the reason too, I looked at the code base of the monopod and nothing looked out of the blue yet, I used the most recent build of it and yet I only get kyverno-cli in README

Reply:
I think this is mostly non-blocking and as per my discussion with @imjasonh on Slack, we can tackle this separately and add it later, if that sounds great!